index.js

```diff
@@ -7,7 +7,12 @@ var Speaker = require('speaker');
 
 var audioPath = path.join(__dirname, 'audio/Breakfast-Machine.mp3');
 
-fs.createReadStream(audioPath)
-  .pipe(new lame.Decoder())
-  .on('format', console.log)
-  .pipe(new Speaker());
+function playLoop() {
+  fs.createReadStream(audioPath)
+    .pipe(new lame.Decoder())
+    .on('format', console.log)
+    .pipe(new Speaker())
+    .on('finish', playLoop);
```

Comment:
Did you test this? AFAIK speaker doesn't have a "finish" event as it is always running, this is why I ended up using `pump`.

If this worked then that's rad and I'd likely still land it